gradle.properties

```diff
@@ -1,4 +1,4 @@
-ideaVersion = IC-2019.3
+ideaVersion = IU-2019.3
```

Comment:
Is there any reason to run IJ Ultimate instead of community?

Reply:
The Jetbrains kubernetes plugin is not available for the community edition so if we don't change it here the mapping on the `build.gradle` fails as there is no-existing plugin for IC

```
def versionsMap = ['IU-2019.2':'192.5728.98','IU-2019.3':'193.5233.144','IU-2020.1':'201.6668.99', 'IU-2020.2':'202.6397.93', 'IU-2020.3':'203.5981.152']

'com.intellij.kubernetes:' + versionsMap[ideaVersion]
```

Reply:
https://plugins.jetbrains.com/plugin/10485-kubernetes/versions